### PR TITLE
Issue 2269 - remove hzn_bash_autocomplete.sh from the bash configure files

### LINKED ISSUE #2269: horizon-cli-uninstall.sh on macOS should remove hzn_bash_autocomplete.sh that agent-install.sh put in .bashrc and .bash_profile
`agent-install.sh` adds `source /usr/local/share/horizon/hzn_bash_autocomplete.sh` to `~/.bashrc` and `~/.bash_profile`. (See https://github.com/open-horizon/anax/blob/4e501e7ed05da435ece7603996483e429c5393de/agent-install/agent-install.sh#L1593 .) So `horizon-cli-uninstall.sh` should remove those lines, if they exist. Otherwise the sourcing of those files will give an error.

### PR DESCRIPTION
**Pull request by xauthulei**
@bmpotter Would you review this, I want try to fix #2269 , thanks

### CODE REVIEW

**Review by dabooz — CHANGES_REQUESTED** [review]
Please update the PR title (which comes from the commit message) to conform to "Issue xxxx - some description". thanks

**Comment by xauthulei** [comment]
> Please update the PR title (which comes from the commit message) to conform to "Issue xxxx - some description". thanks

@dabooz , I have updated it, please review this again, many thanks

File: pkg/mac/scripts/horizon-cli-uninstall.sh [hunk 1]
```diff
@@ -31,3 +31,14 @@ rm -f $DESTDIR/share/man/*/man1/hzn.1
 echo "Removing $SRCDIR/horizon-cli ..."
 rm -rf $SRCDIR/horizon-cli
 
+# remove hzn_bash_autocomplete.sh that agent-install.sh put in .bashrc and .bash_profile
+if [ -f "${HOME}/.bash_profile" ]; then
+        if [[ "A`grep 'hzn_bash_autocomplete' ~/.bash_profile`" != "A"  ]];then
```
Comment (bmpotter) [inline pkg/mac/scripts/horizon-cli-uninstall.sh:36]: Why do you have to check this with `grep`? The sed command will do nothing if the autocomplete script is not found. Same comment applies to `.bashrc` below.

File: pkg/mac/scripts/horizon-cli-uninstall.sh [hunk 2]
```diff
@@ -31,3 +31,14 @@ rm -f $DESTDIR/share/man/*/man1/hzn.1
...
+                sed -i "" "/hzn_bash_autocomplete/d" ~/.bash_profile
```
Comment (bmpotter) [inline pkg/mac/scripts/horizon-cli-uninstall.sh:37]: The full script name is `hzn_bash_autocomplete.sh`, please use that. Same comment applies to .bashrc below.

**Review by bmpotter — CHANGES_REQUESTED** [review]

File: pkg/mac/scripts/horizon-cli-uninstall.sh (hunk 1 above)
Comment (xauthulei) [reply pkg/mac/scripts/horizon-cli-uninstall.sh:36]: @bmpotter , You are right, It sounds to be a little complicated. I will remove this condition. Thanks

**Comment by xauthulei** [comment]
@bmpotter ,Please review this again. Many thanks

**Review by bmpotter — APPROVED** [review]

**Comment by xauthulei** [comment]
@dabooz , could you help review this , Thanks

**Review by dabooz — APPROVED** [review]

**Comment by xauthulei** [comment]
@dabooz  and @linggao , Thanks

**Pull request merged**